python/dolfinx/io/gmshio.py

```diff
@@ -319,4 +323,5 @@ def read_from_msh(filename: str, comm: _MPI.Comm, rank: int = 0, gdim: int = 3,
                       11: ("tetrahedron", 2), 12: ("hexahedron", 2),
                       15: ("point", 0), 21: ("triangle", 3),
                       26: ("interval", 3), 29: ("tetrahedron", 3),
-                      36: ("quadrilateral", 3)}
+                      36: ("quadrilateral", 3),
+                      92: ("hexahedron", 3)}
```

Comment:
Is this tested? I don't think we have the dof permutations for it, ref https://github.com/FEniCS/dolfinx/blob/main/cpp/dolfinx/io/cells.cpp#L294-L303

Reply:
Later in the PR I see you have added a fixme to support higher order hexes.

Reply:
It now throws and exception at a place that makes the source of the error/limitation clear.